test/math.jl

```diff
@@ -387,12 +387,20 @@ j33 = besselj(3,3.)
 @test besselj(-3,-3) == j33
 @test besselj(-3,3) == -j33
 @test besselj(3,-3) == -j33
+@test besselj(3,3f0) ≈ j33
```

Comment:
In that case, it's up to the libm/openspecfun (there are 3 different C functions involved)